CHANGELOG.md

```diff
@@ -8,14 +8,264 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## [5.0.0]
 
+### Added
+
+- Average slam rate to json slam report ([#3080])
```

Comment:
Can you change `Average` to some verb that describes the change to the slam rate?

Reply:
reworded